### PR TITLE
Update Substra logo

### PR DESCRIPTION
**Pull request by natct10**
Hi, this PR just updates the Substra logo used on the README with the new design!
Please let me know if the size or format needs to be updated.

### CODE REVIEW

**Review by jmorel — APPROVED** [review]

**Comment by jmorel** [comment]
BTW do you know when the new logo will be available as a proper svg for integration into the frontend?

**Comment by Kelvin-M** [comment]
We should preserve the same extension for the logo no ? `.svg` instead of using a `.png` file

**Comment by natct10** [comment]
> We should preserve the same extension for the logo no ? `.svg` instead of using a `.png` file

Agreed, but I am not sure that we have this one in `svg`, @ClementMayer do you about this?

**Comment by jmorel** [comment]
This SVG isn't exactly a SVG and that's why I didn't include it in the frontend yet: it a SVG that includes a JPEG inside, which makes it lose the interesting properties of SVG. It fine for this repo, but not for the frontend.

**Review by Kelvin-M — DISMISSED** [review]

**Comment by jmorel** [comment]
Could you update the readme so that the logo isn't gigantic when displayed on github?

**Review by jmorel — CHANGES_REQUESTED** [review]
Could you update the readme so that the logo isn't gigantic when displayed on github?

**Comment by natct10** [comment]
> Could you update the readme so that the logo isn't gigantic when displayed on github?

I know! I was looking for a way to set a size in markdown or to resize the file

**Review by Esadruhn — COMMENTED** [review]
For the size in the README, does using HTML work ?

`drawing`

**Comment by natct10** [comment]
> For the size in the README, does using HTML work ?
> 
> `drawing`

Yes it should work, I'll try this, thank you @Esadruhn

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ![Substra](./substra-logo.svg)
+<img src="substra-logo.svg" alt="drawing" width="200"/>
```
Comment (jmorel) [inline README.md:1]: Could you add the `<h1> ` around the logo again? And change the alt from "drawing" to "substra" ? Thanks!

**Review by jmorel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (natct10) [reply README.md]: Sure, thank you for this insight @jmorel!
Done

**Review by jmorel — APPROVED** [review]

**Pull request merged**